<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCallSetting</class>
 <widget class="QWidget" name="VideoCallSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>510</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoCallSetting{
	border:1px solid #E5E8EF;
}

#content_widget{
	border-top:1px solid #E5E8EF;
	border-bottom:1px solid #E5E8EF;
}

*{
	font-family : 'Microsoft YaHei';
	font-size:12px;
    color:#4E5969;
	outline:0px;
}

QWidget{
	background:#fff;
}


#lbl_title{
	font-size: 14px;
	color: #1D2129;
}

#lbl_kbps{
	color: #1D2129;
}

#btn_cancel{
background: #F2F3F8;
border-radius: 2px;
}

#btn_confirm{
	color:#fff;
	background: #1664FF;
	border-radius: 2px;
}


QComboBox {
  color: #1D2129;
  background: #F2F3F8;
  border-radius: 2px;
  border-radius : 4px;
  padding-left : 16px;
  padding-top : 5px;
  padding-bottom : 5px;
}

QComboBox::drop-down {
  margin-right : 10px;
  border:  none;
}

QComboBox::down-arrow {
  width:16px;
  height: 16px;
  background-image: url(:img/down);
  background-position:center;
  background-repeat:no-repeat;
}

QComboBox::ListView{
    outline: 1px solid rgba(0,0,0,0);  

}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="top_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>46</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_title">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_close{border: none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_close</normaloff>:/img/videocall_close</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>428</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>428</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_resolutions">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>0</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_resolution">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_quality">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>0</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_quality">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_mirror_camera">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CheckButton" name="mirror_camera_btn" native="true">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckButton</class>
   <extends>QWidget</extends>
   <header>check_button.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/video_call_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallSetting</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_confirm</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallSetting</receiver>
   <slot>onConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>854</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallSetting</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClose()</slot>
  <slot>onHistoryShow()</slot>
  <slot>onRecordShow()</slot>
  <slot>onConfirm()</slot>
  <slot>onCancel()</slot>
 </slots>
</ui>
